{{cookiecutter.project_slug}}/.circleci/config.yml

```diff
@@ -80,11 +80,16 @@ jobs:
             bundle exec fastlane test
             mv fastlane/report.xml test-results/fastlane
 
-      - store_test_results:
-          path: test-results
+      #- store_test_results:
+      #    path: test-results
 
-      - store_artifacts:
-          path: test-results
+      #- store_artifacts:
+      #    path: test-results
```

Comment:
We're not using the test results for anything so I commented these out to save some build time. If someone needs them in their project they can uncomment them.

Reply:
gotcha